components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -140,7 +141,8 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
          tracersTendPool,   &! structure holding tracer tendencies
          forcingPool,       &! structure holding forcing variables
          scratchPool,       &! structure holding temporary variables
-         swForcingPool       ! structure holding short-wave forcing vars
+         swForcingPool,     &! structure holding short-wave forcing vars
+         diagnosticsPool     ! structure holding diagnostic (not tendency) variables
```

Comment:
@pbosler, it looks like this change is causing conflicts with other recent changes to `master`.  But it also looks like this pool is being retrieved by not actually used for anything.  Could you take it back out (also on line 304) so the conflict will be resolved?